### PR TITLE
Update ElkBot

### PR DESCRIPTION
**Pull request by rivques**

### CODE REVIEW

File: RLBotPack/ElkBot/ExampleBot.cfg [hunk 1]
```diff
@@ -28,5 +28,3 @@ github = https://github.com/rivques/ElkBot
 
 # Programming language
 language = Python
-
-tags = 1v1, teamplay
```
Comment (VirxEC) [inline RLBotPack/ElkBot/ExampleBot.cfg:32]: please add this back

**Review by VirxEC — CHANGES_REQUESTED** [review]
changes needed

File: RLBotPack/ElkBot/ExampleBot.cfg (hunk 1 above)
Comment (rivques) [reply RLBotPack/ElkBot/ExampleBot.cfg:32]: oh yeah right

**Comment by VirxEC** [comment]
nice

**Review by VirxEC — APPROVED** [review]

**Pull request merged**